### PR TITLE
Replacing NameVal > ChatManager & Adding WorldEdit

### PR DESCRIPTION
**Pull request by Rozen4334**
Take 2

### CODE REVIEW

**Comment by Rozen4334** [comment]
Updated rapidly as a few minor functions were unavailable

File: README.md [hunk 1]
```diff
@@ -10,6 +10,12 @@ This is the TShock plugin repository. It collects links to all plugins in the ec
 ----
 
 ## List of all plugins! 🧪☕️⚡️
+* [ChatManager](https://github.com/Rozen4334/ChatManager/releases/tag/v1.1) by Rozen4334
+  * A chat & username checker plugin that offers lots of configuration & lots of features.
+  * Tested on TShock 4.5.5.
+  * NOTE: Please remove NameValidator as this replaces & upgrades that plugin.
+  * [Download](https://files.catbox.moe/wt27p8.zip)
```
[the remark this answers is not in the document]
Comment (hakusaro) [reply README.md:17]: pf ha: https://hybrid-analysis.com/sample/508faa741eaab64a26c33137efa228d5ac79ed56439e0a70b033686c2b503d97

sha256:  508faa741eaab64a26c33137efa228d5ac79ed56439e0a70b033686c2b503d97 

```
~/Downloads via 🐍 v2.7.16 via 💎 v3.0.1
❯ shasum -a 256 /Users/particles/Downloads/zwjwlh/ProfanityFilter.dll
508faa741eaab64a26c33137efa228d5ac79ed56439e0a70b033686c2b503d97  /Users/particles/Downloads/zwjwlh/ProfanityFilter.dll

~/Downloads via 🐍 v2.7.16 via 💎 v3.0.1
❯ shasum -a 256 /Users/particles/Downloads/wt27p8/ProfanityFilter.dll
508faa741eaab64a26c33137efa228d5ac79ed56439e0a70b033686c2b503d97  /Users/particles/Downloads/wt27p8/ProfanityFilter.dll
```

(not reviewed again)

[the remark this answers is not in the document]
Comment (hakusaro) [reply README.md:17]: Screen Shot 2021-08-11 at 23 01 56

for posterity

**Review by hakusaro — APPROVED** [review]
lgtm_trick

**Pull request merged**